proto/orc_proto.proto

```diff
@@ -386,6 +386,7 @@ enum CompressionKind {
   LZO = 3;
   LZ4 = 4;
   ZSTD = 5;
+  BROTLI = 6;
```

Comment:
In order to check this in, we'd better update the website and the C++ code (at least throw an unimplemented exception on the reader side)

Reply:
Thanks @wgtmac I will do it.